src/Rules/Library/ChromiumComponentsShouldUseWebScanner.cs

```diff
@@ -6,14 +6,13 @@
 using Axe.Windows.Rules.Resources;
 using static Axe.Windows.Rules.PropertyConditions.ControlType;
 using static Axe.Windows.Rules.PropertyConditions.Framework;
-using static Axe.Windows.Rules.PropertyConditions.Relationships;
 
 namespace Axe.Windows.Rules.Library
 {
     [RuleInfo(ID = RuleId.ChromiumComponentsShouldUseWebScanner)]
     class ChromiumComponentsShouldUseWebScanner : Rule
     {
-        public ChromiumComponentsShouldUseWebScanner()
```

Comment:
I could build an argument for renaming this to `ChromiumDocumentsShouldUseWebScanner`, but that would make the PR harder to review. Happy to put this into a subsequent PR if needed.